src/exporter/components/export-table/cell-renderer/progress.cell-renderer.tsx

```diff
@@ -3,13 +3,16 @@ import { ICellRendererParams } from 'ag-grid-community';
 import { LinearProgress, Typography } from '@map-colonies/react-core';
 import { Box } from '@map-colonies/react-components';
 import './progress.cell-renderer.css';
+import { useState } from 'react';
+import EXPORTER_CONFIG from '../../../../common/config';
 import { IExportTaskStatus } from '../../../models/exportTaskStatus';
 
 const PEDING_PROGRESS = 0;
 const ONE_HUNDRED = 100;
 export const ProgressRenderer: React.FC<ICellRendererParams> = (
   props
 ) => {
+  const [isReversed] = useState<boolean>(EXPORTER_CONFIG.I18N.DEFAULT_LANGUAGE === 'he' ? true : false);
```

Comment:
isReversed or isRtl -> use always isRTL (don't use different names for the same meaning)

Reply:
fixed in dbef340